src/worker/jobs/rss/sync_updates_feed.rs

```diff
@@ -0,0 +1,139 @@
+use crate::schema::{crates, versions};
+use crate::storage::FeedId;
+use crate::worker::Environment;
+use anyhow::anyhow;
+use crates_io_worker::BackgroundJob;
+use diesel::prelude::*;
+use std::sync::Arc;
+
+#[derive(Serialize, Deserialize)]
+pub struct SyncUpdatesFeed;
+
+const NUM_ITEMS: i64 = 100;
+
+impl BackgroundJob for SyncUpdatesFeed {
+    const JOB_NAME: &'static str = "sync_updates_feed";
+
+    type Context = Arc<Environment>;
+
+    async fn run(&self, ctx: Self::Context) -> anyhow::Result<()> {
+        let feed_id = FeedId::Updates;
+        let domain = &ctx.config.domain_name;
+
+        info!("Loading latest {NUM_ITEMS} version updates from the database…");
+        let conn = ctx.deadpool.get().await?;
+        let version_updates = conn
+            .interact(load_version_updates)
+            .await
+            .map_err(|err| anyhow!(err.to_string()))??;
+
+        let link = rss::extension::atom::Link {
+            href: ctx.storage.feed_url(&feed_id),
+            rel: "self".to_string(),
+            mime_type: Some("application/rss+xml".to_string()),
+            ..Default::default()
+        };
+
+        let items = version_updates
+            .into_iter()
+            .map(|u| u.into_rss_item(domain))
+            .collect();
+
+        let channel = rss::Channel {
+            title: "crates.io: recent updates".to_string(),
+            link: format!("https://{domain}/"),
+            description: "Recent version publishes on the crates.io package registry".to_string(),
+            language: Some("en".to_string()),
+            atom_ext: Some(rss::extension::atom::AtomExtension { links: vec![link] }),
+            items,
+            ..Default::default()
+        };
```

Comment:
We should probably define an XML namespace here:

```suggestion
        let channel = rss::Channel {
            title: "crates.io: recent updates".to_string(),
            link: format!("https://{domain}/"),
            description: "Recent version publishes on the crates.io package registry".to_string(),
            language: Some("en".to_string()),
            atom_ext: Some(rss::extension::atom::AtomExtension { links: vec![link] }),
            namespaces: [(
                "crates".to_string(),
                "https://crates.io/".to_string(),
            )],
            items,
            ..Default::default()
        };
```

(pick whatever opaque URL-y string you want for the namespace, as long as we stick to it)

Reply:
done